anesthetic/samples.py

```diff
@@ -113,132 +116,6 @@ def _set_automatic_limits(self):
             if param not in self.limits:
                 self.limits[param] = (self[param].min(), self[param].max())
 
-    def plot(self, ax, paramname_x, paramname_y=None, *args, **kwargs):
-        """Interface for 2D and 1D plotting routines.
-
-        Produces a single 1D or 2D plot on an axis.
-
-        Parameters
-        ----------
-        ax: matplotlib.axes.Axes
-            Axes to plot on
-
-        paramname_x: str
-            Choice of parameter to plot on x-coordinate from self.columns.
-
-        paramname_y: str
-            Choice of parameter to plot on y-coordinate from self.columns.
-            optional, if not provided, or the same as paramname_x, then 1D plot
-            produced.
-
-        plot_type: str
-            Must be in {'kde', 'scatter', 'hist', 'fastkde'} for 2D plots and
-            in {'kde', 'hist', 'fastkde', 'astropyhist'} for 1D plots.
-            optional, (Default: 'kde')
```

Comment:
This should now be called `kind` and put into `plot_1d` docstring.

Reply:
Updated in 0a3b7e0